acapy_agent/admin/tests/test_admin_server.py

```diff
@@ -112,161 +103,7 @@ async def test_ready_middleware(self):
             with self.assertRaises(KeyError):
                 await test_module.ready_middleware(request, handler)
 
-    async def test_ready_middleware_http_unauthorized(self):
-        """Test handling of web.HTTPUnauthorized and related exceptions."""
-        with mock.patch.object(test_module, "LOGGER", mock.MagicMock()) as mock_logger:
-            mock_logger.info = mock.MagicMock()
-
```

Comment:
Seems that the rebase dropped some of the unit tests I added :-) from #3327 
I'll re-add in follow-up PR 👍